holmes/plugins/toolsets/grafana/loki_api.py

```
@@ -65,10 +65,19 @@ def _make_request():
         return []
-
-    except requests.exceptions.RequestException as e:
-        raise Exception(f"Failed to query Loki logs: {str(e)}")
+    except Exception as e:
+        raw = response.text
+        raise Exception(
+            f"Failed to process Loki response: {e}\n"
+            f"--- raw response ({len(raw)} chars, content-type={response.headers.get('Content-Type', 'unknown')}) ---\n"
+            f"{raw}\n"
+            f"--- end raw response ---"
+        )
```

Ruff (0.15.12)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)